model/src/pyrenew/model/hospitalizations.py

```diff
@@ -76,23 +78,24 @@ def __init__(
     @staticmethod
     def validate(latent_hospitalizations, observed_hospitalizations) -> None:
         _assert_sample_and_rtype(latent_hospitalizations, skip_if_none=False)
-        _assert_sample_and_rtype(observed_hospitalizations, skip_if_none=True)
+        _assert_sample_and_rtype(observed_hospitalizations, skip_if_none=False)
         return None
 
     def sample_hospitalizations_latent(
         self,
-        random_variables: dict,
-        constants: dict = None,
+        infections: ArrayLike,
+        **kwargs,
     ) -> tuple:
         return self.latent_hospitalizations.sample(
-            random_variables=random_variables,
-            constants=constants,
+            latent=infections,
+            **kwargs,
         )
 
     def sample_hospitalizations_obs(
```

Comment:
Needs updated docstring (still refers to `constants` and `random_variables`)

Reply:
You are commenting on the deleted code.

Reply:
Oops, sorry!

Reply:
This was addressed in 44cb3d9